packages/@sanity/vision/src/SUPPORTED_PERSPECTIVES.ts

```diff
@@ -0,0 +1,15 @@
+import {type ClientPerspective} from '@sanity/client'
+
+export type SupportedPerspective = 'raw' | 'previewDrafts' | 'published' | 'drafts'
+
+export const SUPPORTED_PERSPECTIVES = [
+  'raw',
+  'previewDrafts',
+  'published',
+  'drafts',
```

Comment:
Does something changes from `previewDrafts` to `drafts`?  I think they will return the same

Reply:
yep, they are equivalent